ui/src/app/components/search-input.tsx

```diff
@@ -89,7 +89,20 @@ export interface SearchInputProps {
   onChange: (newInput: string) => void;
 }
 
-export class SearchInput extends React.Component<SearchInputProps, SearchInputState> {
+export class SearchInput extends React.Component<any, SearchInputState> {
+  static defaultProps = {
```

Comment:
is defaultProps referenced anywhere?

Reply:
Good question. It's referenced by the React framework when it renders this component.